javascript/javascript1/week3/SeriesdurationOfmylife.js

```diff
@@ -0,0 +1,45 @@
+ // Seies duration of my life. 
+
+ const seriesDurations = [
+    {
+      title: "CSI",
+      days: 6,
+      hours: 18,
+      minutes: 24,
+    },
+    {
+      title: "brotherhood",
+      days: 1,
+      hours: 5,
+      minutes: 0,
+    },
+    {
+      title: "Halo",
+      days: 1,
+      hours: 14,
+      minutes: 0,
+    },
+  ];
+  
+  function logOutSeriesText() {
+    
+     let age = 80; 
+     let ageToMinutes= 80 * 365 * 24 * 60;
+
+     let totalDuration= 0; 
+     for ( let i = 0; i < seriesDurations.length ; i++) {
```

Comment:
Nice.
Well structured. 
whole expression is cut into small parts and results are in variables with meaningful names. 
Easy to read.